generators/gae/templates/app.yaml.ejs

```diff
@@ -0,0 +1,20 @@
+runtime: java11
```

Comment:
Maybe start this file with the following:
```
# ========================================================================
# App Engine Configuration.
# Full reference is available at:
# https://cloud.google.com/appengine/docs/flexible/java/reference/app-yaml
# ========================================================================

Reply:
Thanks. Done 😄